### ISSUE TITLE
Launchpad Utility website domain expired

### ISSUE THREAD

**Issue by Tralen** [mat1jaczyyy/lpp-performance-cfw#61]
Hello,

The domain for the Launchpad Utility has expired and the address now links to an unsafe website.

Is there another way of accessing the tool or the will domain be renewed?

Thank you.

**Comment by mat1jaczyyy** [comment]
yeah, i noticed. will deal with that monday. https://lp-firmware-utility.netlify.app/ is temporarily available
